json/tbd_PSI_combo.json

```diff
@@ -7,8 +7,7 @@
       "rimjoist": 0.7,
       "parapet": 0.7,
       "fenestration": 0.7,
-      "concave": 0.7,
-      "convex": 0.7,
+      "corner": 0.7,
```

Comment:
Published PSI data on _concave_ vs _convex_ variants of a given thermal bridge type (such as corners) remains sparse. Awaiting future published data, the solution here is nonetheless generalized to handle _flat_, _concave_ or _convex_ variants of any thermal bridge type/edge ... with the exception of joints and transitions (discussed a bit later on). Variants may inherit parent PSI values (e.g., _corner_).